src/qt/transactionview.h

```diff
@@ -70,6 +70,7 @@ class TransactionView : public QWidget
     QComboBox *watchOnlyWidget;
     QLineEdit *search_widget;
     QLineEdit *amountWidget;
+    QDoubleValidator *amountValidator;
```

Comment:
This isn't used anywhere. Probably we either need to delete it in cleanup, or don't need to store it at all.

Reply:
it is used: it's used to validate what is entered in the amount field

Reply:
Oh I agree with @luke-jr actually; there's no need to store this in the object, just store it temporarily in a local variable in `TransactionView::TransactionView` while you need to manipulate it.